packages/amplication-client/src/VersionControl/Build.tsx

```diff
@@ -31,6 +32,7 @@ const Build = ({ build, onError, open }: Props) => {
       <PanelHeader>Build details</PanelHeader>
       <div className={`${CLASS_NAME}__message`}>{data.build.message}</div>
       <UserAndTime account={account} time={build.createdAt} />
+      <BuildHeader build={data.build} isError={open} />
```

Comment:
I tried to put the BuildHeader component back but then I got an error that I must pass the isError prop, but before I removed it we didn't pass this prop

Reply:
what is `open`? maybe you can just use "false"